assignments/objects.js

```diff
@@ -19,27 +19,81 @@ const example = {
 
 // Write your intern objects here:
 
+const student1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
```

Comment:
you could have also done something like this and then I will show you the console.log at the end
let mitzi = {
  "id": 1,
  "name": "Mitzi",
  "email": "mmelloy0@psu.edu",
  "gender": "F"
}